### PR TITLE
[JMusicBot] QoL Update & Fixes

### PR DESCRIPTION
**Pull request by lilkingjr1**
__**Egg:**__
- Updated image to use new yolks Java image.
- Removed unused `-Djavax.accessibility.assistive_technologies=" "` flag from startup command.
- Updated and added as many Startup Variables as possible for basic/common settings in the `config.txt` file.
- Updated the install script to (1.) have more verbose/clear logging, (2.) update old `config.txt` files with newer versions (while retaining the old one for posterity at `config.txt-old`), and (3.) moving the `example_playlist.txt` file to the correct directory.

__**`config.txt` File:**__
- Updated to the latest version.

__**`playlist.txt` File:**__
- Renamed to `example_playlist.txt`.
- Updated in general to match Wiki's example.

__**Readme:**__
- Added authors
- Added a better bot description.
- Improved "Configuration" section.
- Added "Updating" section.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### CODE REVIEW

File: bots/discord/jmusicbot/egg-j-music-bot.json [hunk 1]
```diff
@@ -1,58 +1,130 @@
...
             "container": "debian:buster-slim",
-            "entrypoint": "\/bin\/bash"
+            "entrypoint": "bash"
```
Comment (parkervcp) [inline bots/discord/jmusicbot/egg-j-music-bot.json:26]: This should stay `/bin/bash` as poisoning the env vars is still potentially possible.

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: bots/discord/jmusicbot/egg-j-music-bot.json (hunk 1 above)
Comment (lilkingjr1) [reply bots/discord/jmusicbot/egg-j-music-bot.json:26]: Good catch. I was just trying to simplify and match other eggs I've worked on. I'll fix it soon.

**Comment by Software-Noob** [comment]
Please print the variables/URLs for easier troubleshooting. We've been receiving a lot of complaints and no real way to track what's done because most eggs perform silent steps.

For example:

```
echo -e "running curl -sSLo config.txt ${CONFIG_LINK}"
curl -sSLo config.txt ${CONFIG_LINK}

**Comment by lilkingjr1** [comment]
@Software-Noob Good idea; I'll implement that now.

**Comment by lilkingjr1** [comment]
@Software-Noob Implemented in 72765cc

**Review by parkervcp — APPROVED** [review]

**Pull request merged**